cocos/serialization/deserialize.ts

```diff
@@ -481,13 +450,37 @@ interface IFileData extends Array<any> {
     [File.DependUuidIndices]: (StringIndex|string)[];
 }
 
-// type Body = Pick<IFileData, File.Instances | File.InstanceTypes | File.Refs | File.DependObjs | File.DependKeys | File.DependUuidIndices>
-type Shared = Pick<IFileData, File.Version | File.SharedUuids | File.SharedStrings | File.SharedClasses | File.SharedMasks>;
-const PACKED_SECTIONS = File.Instances;
-interface IPackedFileData extends Shared {
-    [PACKED_SECTIONS]: IFileData[];
+type IFileData = MapEnum<{
+    [x in keyof IFileDataMap as `${x}`]: IFileDataMap[x];
+}, 11 /* Currently we should manually specify the enumerators count. */>;
```

Comment:
Our TypeScript version is lower. It can't be:

![image](https://github.com/cocos/cocos-engine/assets/28807867/814eecf7-6aaa-4e74-980f-7ca56da199b8)